### PR TITLE
Make -o option working for rolling-update

### PR DESCRIPTION
**Pull request by hurf**
Fixes: #13443

Use simple message to show the result. If -o is specified, print
updated rc.

### CODE REVIEW

**Comment by bgrant0607** [comment]
cc @kubernetes/kubectl

**Comment by bgrant0607** [comment]
Thanks, @hurf

File: pkg/kubectl/cmd/rollingupdate.go [hunk 1]
```diff
@@ -298,10 +305,14 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
...
-	} else {
-		fmt.Fprintf(out, "%s\n", newRc.Name)
+		newRc.Name = oldName
+	}
+	newRc, err = client.ReplicationControllers(cmdNamespace).Get(newRc.Name)
+	if outputFormat != "" {
+		return f.PrintObject(cmd, newRc, out)
 	}
+	_, resource := meta.KindToResource(newRc.Kind, false)
+	cmdutil.PrintSuccess(mapper, false, out, resource, oldName, "rolling updated to \""+newRc.Name+"\"")
```
Comment (kargakis) [inline pkg/kubectl/cmd/rollingupdate.go]:
```go
fmt.Sprintf("rolling updated to \"%s\"", newRc.Name)
```
instead of concatenating strings

Comment (ironcladlou) [inline pkg/kubectl/cmd/rollingupdate.go]: Further refinement:

```go
fmt.Sprintf("rolling updated to %q", newRc.Name)
```

That'll automatically double-quote `newRc.Name`, if double quotes are what you crave.

Comment (kargakis) [inline pkg/kubectl/cmd/rollingupdate.go]: Right, forgot about %q, use that

File: pkg/kubectl/cmd/rollingupdate.go [hunk 2]
```diff
@@ -298,10 +305,14 @@ func RunRollingUpdate(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, arg
 	}
 
 	if keepOldName {
-		fmt.Fprintf(out, "%s\n", oldName)
-	} else {
-		fmt.Fprintf(out, "%s\n", newRc.Name)
+		newRc.Name = oldName
+	}
+	newRc, err = client.ReplicationControllers(cmdNamespace).Get(newRc.Name)
```
Comment (JanetKuo) [inline pkg/kubectl/cmd/rollingupdate.go]: check the `err` returned here

File: pkg/kubectl/cmd/rollingupdate.go (hunk 1 above)
Comment (JanetKuo) [inline pkg/kubectl/cmd/rollingupdate.go]: if `keepOldName`, should we print `<resource> "A" rolling updated to "A"` or just `<resource> "A" rolling updated`?

**Comment by JanetKuo** [comment]
Nits; otherwise LGTM

**Comment by hurf** [comment]
@kargakis @ironcladlou @JanetKuo - Thanks for the advices, patch updated.

**Comment by JanetKuo** [comment]
Thanks @hurf!

**Pull request merged**